frontend_tests/node_tests/message_list_view.js

```diff
@@ -88,6 +88,31 @@ function build_list(message_groups) {
     return list;
 }
 
+run_test('test_rows', () => {
+    var list = {};
+
+    var view = new MessageListView(list, 'test_table', false);
+    var jquery_rows = {};
+
+    function build_row(id) {
+        jquery_rows[id] = $('#row_' + id);
+        view._rows[id] = '#row_' + id;
+    }
+
+    build_row(1);
+    build_row(2);
+    build_row(3);
+    build_row(3.1);
```

Comment:
What is the point of this? Does adding a 3.1 row break some code/enter a failure codepath?

Reply:
It's probably worth a comment, but decimal here is part of how local echo works.  We should use `3.01`, though, right, since that's the default local echo format?

Reply:
See discussion here, too:

https://chat.zulip.org/#narrow/stream/65-checkins/subject/lonerz/near/611478